afids-auto-train/workflow/Snakefile

```diff
@@ -67,10 +95,27 @@ rule align_template_rigid:
     shell:
         "reg_aladin -flo {input.image} -ref {params.fixed} -res {output.warped} -rigOnly -interp 0 -aff {output.xfm_ras}"
 
-rule gen_grad_template:
+rule normalize: 
     input:
-        image=rules.align_template_rigid.output.warped,
+        im=rules.align_template_rigid.output.warped,
     output:
+        im_norm=bids(
+            root=join(config["output_dir"], "normalized"),
+            datatype="anat",
+            res="1mm",
+            suffix="normT1w.nii.gz",
```

Comment:
I would add `desc="norm"` (or `synthstripnorm`) and leave the suffix as `T1w.nii.gz`